lib/shellable.ts

```diff
@@ -76,6 +76,14 @@ export interface ShellableOptions {
    */
   buildProjectName?: string;
 
+  /**
+   * Indicates if Regional AWS STS endpoints should be used instead
+   * of the global endpoint. Specify true to use Regional AWS STS endpoints.
+   *
+   * @default false
+   */
+  useRegionalSTSEndpoints?: boolean;
```

Comment:
```suggestion
  useRegionalStsEndpoints?: boolean;
```

Reply:
Renamed this member variable and also a local variable `STSEndpoints` -> `StsEndpoints` in  LinuxPlatform.prebuildCommands.